src/recordlinker/schemas/pii.py

```diff
@@ -441,27 +438,31 @@ def feature_iter(self, feature: Feature) -> typing.Iterator[str]:
                     yield str(race)
         elif attribute == FeatureAttribute.TELECOM:
             for telecom in self.telecom:
+                value = telecom.value.strip()
                 if telecom.system is None:
-                    yield telecom.value.strip().lower()
+                    yield value
                     continue
-
-                handlers = Telecom.get_system_handlers()
-
-                if telecom.system in handlers:
-                    value = getattr(telecom, handlers[telecom.system])()
-                    if value:
-                        yield value
-
+                elif telecom.system == "email":
+                    yield value
+                elif telecom.system == "phone":
```

Comment:
I think we just need 1 if/else block and no elif blocks here.  system == None or system == "email" is exactly the same as the else block.

Reply:
Ah you're right; forgot to clean it up after making other changes. Good catch!